partition/bootloader_test.go

```diff
@@ -157,3 +161,23 @@ func (s *PartitionTestSuite) TestInstallBootloaderConfig(c *C) {
 		c.Assert(osutil.FileExists(fn), Equals, true)
 	}
 }
+
+func (s *PartitionTestSuite) TestRebootNoError(c *C) {
+	systemdCmd := testutil.MockCommand(c, "shutdown", `
+exit 0
+	`)
+	defer systemdCmd.Restore()
+
+	err := reboot(10)
+	c.Assert(err, IsNil)
```

Comment:
I think we also want to `c.Check(systemdCmd.Calls(), DeepEquals, [][]string{ {"shutdown", "+10"} })` (or similar) here just to ensure that a) shutdown got really called b) it was called with the right parameters.